src/main/java/com/nirima/jenkins/plugins/docker/DockerNodeFactory.java

```diff
@@ -0,0 +1,106 @@
+package com.nirima.jenkins.plugins.docker;
+
+import com.google.inject.internal.asm.$ClassVisitor;
+import com.nirima.jenkins.plugins.docker.cloudstat.CloudStatsFactory;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.model.Descriptor;
+import hudson.model.Node;
+import hudson.slaves.ComputerLauncher;
+import hudson.slaves.NodeProvisioner;
+import io.jenkins.docker.DockerTransientNode;
+import org.jenkinsci.plugins.cloudstats.ProvisioningActivity;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.concurrent.Future;
+
+public interface DockerNodeFactory extends Serializable  {
```

Comment:
Whitespace: should be one (and only one) space before the `{`

Later lines have similar issues w.r.t. inconsistent whitespace.